### DISCUSSION TITLE
Black screenshots in RT Half-Life - any fix?

### DISCUSSION THREAD

**Discussion by AdamskiKamil** [sultim-t/xash-rt#86] [Q&A]
Hi! 
I'm having issues with the built-in screenshot functionality. I'm running the game with the latest version (build 1.0.5), native resolution (QHD), and no image reconstruction method enabled (although downloaded and installed DLSS package since I own nVidia card in hope it might fix it, no luck).

Taken screenshots are the same 15KB files, black void and nothing else. I restarted the game a few times but nothing changed, and there are not a lot of video mode options to tweak in order to somehow fix it.

Screenshots taken with Print Screen are totally fine, but in the long run, it's not the most optimal and convenient way to go.
Any idea how to fix it?

**Comment by sultim-t** [comment]
Screenshots are not supported. But you can use tools like Win+PrintScrn or Geforce Experience to take screenshots
